include/z64item.h

```diff
@@ -377,6 +377,127 @@ typedef enum {
+    /* 0x17 */ DI_BOMB_BAG_20,
+    /* 0x18 */ DI_BOMB_BAG_30,
+    /* 0x19 */ DI_BOMB_BAG_40,
+    /* 0x1A */ DI_STICK,
+    /* 0x1B */ DI_DUNGEON_MAP,
+    /* 0x1C */ DI_SHIELD_DEKU,
+    /* 0x1D */ DI_MAGIC_SMALL,
+    /* 0x1E */ DI_MAGIC_LARGE,
+    /* 0x1F */ DI_BOMB,
+    /* 0x20 */ DI_STONE_OF_AGONY,
+    /* 0x21 */ DI_WALLET_ADULT,
+    /* 0x22 */ DI_WALLET_GIANT,
+    /* 0x23 */ DI_GERUDO_CARD,
+    /* 0x24 */ DI_ARROWS_SMALL,
+    /* 0x25 */ DI_ARROWS_MEDIUM,
+    /* 0x26 */ DI_ARROWS_LARGE,
+    /* 0x27 */ DI_BOMBCHU,
+    /* 0x28 */ DI_EGG,
+    /* 0x29 */ DI_SCALE_SILVER,
+    /* 0x2A */ DI_SCALE_GOLDEN,
+    /* 0x2B */ DI_SHIELD_HYLIAN,
+    /* 0x2C */ DI_HOOKSHOT,
+    /* 0x2D */ DI_LONGSHOT,
+    /* 0x2E */ DI_OCARINA_TIME,
+    /* 0x2F */ DI_MILK,
+    /* 0x30 */ DI_MASK_KEATON,
+    /* 0x31 */ DI_MASK_SPOOKY,
+    /* 0x32 */ DI_SLINGSHOT,
+    /* 0x33 */ DI_BOOMERANG,
+    /* 0x34 */ DI_BOW,
+    /* 0x35 */ DI_LENS,
+    /* 0x36 */ DI_POTION_GREEN,
+    /* 0x37 */ DI_POTION_RED,
+    /* 0x38 */ DI_POTION_BLUE,
+    /* 0x39 */ DI_SHIELD_MIRROR,
+    /* 0x3A */ DI_LETTER_ZELDA,
+    /* 0x3B */ DI_TUNIC_GORON,
+    /* 0x3C */ DI_TUNIC_ZORA,
+    /* 0x3D */ DI_BEAN,
+    /* 0x3E */ DI_FISH,
+    /* 0x3F */ DI_SAW,
+    /* 0x40 */ DI_HAMMER,
+    /* 0x41 */ DI_GRASS,
+    /* 0x42 */ DI_SWORD_BGS,
+    /* 0x43 */ DI_CHICKEN,
+    /* 0x44 */ DI_LETTER_RUTO,
+    /* 0x45 */ DI_OCARINA_FAIRY,
+    /* 0x46 */ DI_BOOTS_IRON,
+    /* 0x47 */ DI_SEEDS,
+    /* 0x48 */ DI_GAUNTLETS_SILVER,
+    /* 0x49 */ DI_GAUNTLETS_GOLD,
+    /* 0x4A */ DI_NCOIN_YELLOW,     // unused
+    /* 0x4B */ DI_NCOIN_RED,        // unused
+    /* 0x4C */ DI_NCOIN_GREEN,      // unused
+    /* 0x4D */ DI_NCOIN_BLUE,       // unused
+    /* 0x4E */ DI_MASK_SKULL,
+    /* 0x4F */ DI_MASK_BUNNY,
+    /* 0x50 */ DI_MASK_TRUTH,
+    /* 0x51 */ DI_EYEDROPS,
+    /* 0x52 */ DI_ODD_POTION,
+    /* 0x53 */ DI_ODD_MUSHROOM,
+    /* 0x54 */ DI_CLAIM_CHECK,
+    /* 0x55 */ DI_SWORD_BROKEN,
+    /* 0x56 */ DI_PRESCRIPTION,
+    /* 0x57 */ DI_BRACELET,
+    /* 0x58 */ DI_SOLDOUT,
+    /* 0x59 */ DI_FROG,
+    /* 0x5A */ DI_MASK_GORON,
+    /* 0x5B */ DI_MASK_ZORA,
+    /* 0x5C */ DI_MASK_GERUDO,
+    /* 0x5D */ DI_COJIRO,
+    /* 0x5E */ DI_BOOTS_HOVER,
+    /* 0x5F */ DI_ARROW_FIRE,
+    /* 0x60 */ DI_ARROW_ICE,
+    /* 0x61 */ DI_ARROW_LIGHT,
+    /* 0x62 */ DI_SKULL_TOKEN,
+    /* 0x63 */ DI_DINS_FIRE,
+    /* 0x64 */ DI_FARORES_WIND,
+    /* 0x65 */ DI_NAYRUS_LOVE,
+    /* 0x66 */ DI_BLUE_FIRE,
+    /* 0x67 */ DI_BUG,
+    /* 0x68 */ DI_BUTTERFLY,        // unused
+    /* 0x69 */ DI_POE,
+    /* 0x6A */ DI_FAIRY,
+    /* 0x6B */ DI_BULLET_BAG,
+    /* 0x6C */ DI_RUPEE_GREEN,
+    /* 0x6D */ DI_RUPEE_BLUE,
+    /* 0x6E */ DI_RUPEE_RED,
+    /* 0x6F */ DI_BIG_POE,
+    /* 0x70 */ DI_RUPEE_PURPLE,
+    /* 0x71 */ DI_RUPEE_GOLD,
+    /* 0x72 */ DI_BULLET_BAG_50,
+    /* 0x73 */ DI_SWORD_KOKIRI,
+    /* 0x74 */ DI_SKULL_TOKEN_2,
+    /* 0x75 */ DI_MAX
+} GetItemDrawID;
```

Comment:
I think it could be better to go with `GID_` for the prefix in this enum, for "Get Item Draw"